sources/HUBViewControllerImplementation.m

```diff
@@ -392,7 +392,8 @@ - (CGRect)frameForBodyComponentAtIndex:(NSUInteger)index
     }
     
     NSIndexPath * const indexPath = [NSIndexPath indexPathForItem:(NSInteger)index inSection:0];
-    return [self.collectionView layoutAttributesForItemAtIndexPath:indexPath].frame;
+    HUBCollectionViewLayout * const layout = (HUBCollectionViewLayout *)self.collectionView.collectionViewLayout;
```

Comment:
Ah, I understand. You should be able to get rid of the casting sine you're just calling a `UICollectionViewLayout` API.